File: CSharp/SpaceVIL/SpaceVIL/Core/SpaceVILException.cs
using System;

namespace SpaceVIL.Core
{
    /// <summary>
    /// SpaceVILException is a class of exceptions of the SpaceVIL.
    /// </summary>
    [Serializable]
    public class SpaceVILException : Exception
    {
        public SpaceVILException() { }
        public SpaceVILException(string message) : base(message) { }
    }
}

File: Java/SpaceVIL/spacevil/src/main/java/com/spvessel/spacevil/Exceptions/SpaceVILException.java
package com.spvessel.spacevil.Exceptions;

/**
 * SpaceVILException is a class of exceptions of the SpaceVIL.
 */
public class SpaceVILException extends RuntimeException {
    private static final long serialVersionUID = 1L;

    // static final long serialVersionUID = 0;
    public SpaceVILException() { }

    public SpaceVILException(String message) {
        super("SpaceVILException: " + message);
    }
}